### PR TITLE
Add french source (Manga-LEL)

### PR DESCRIPTION
**Pull request by Hugome**
Add of one French source (http://manga-lel.com/)

Baguette ! :fr:

### CODE REVIEW

**Comment by M2ck** [comment]
Finally a french source, thanks! 
Possible to add Scantrad (https://scantrad.fr/)?

**Comment by null-dev** [comment]
There is already an active PR for this source: #103 . *oops*

File: src/fr/mangalel/src/eu/kanade/tachiyomi/extension/fr/mangalel/Mangalel.kt [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+import java.security.cert.X509Certificate
+import java.text.ParseException
+import java.text.SimpleDateFormat
+import java.util.*
+import javax.net.ssl.SSLContext
+import javax.net.ssl.X509TrustManager
+
+class Mangalel : ParsedHttpSource() {
+
+    override val id: Long = 11
```
Comment (inorichi) [inline src/fr/mangalel/src/eu/kanade/tachiyomi/extension/fr/mangalel/Mangalel.kt]: Remove this line. Id should be automatically asigned

File: src/fr/mangalel/src/eu/kanade/tachiyomi/extension/fr/mangalel/Mangalel.kt [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+    override val lang = "fr"
+
+    override val supportsLatest = true
+
+    private val catalogHeaders = Headers.Builder().apply {
+        add("User-Agent", "Mozilla/5.0 (Windows NT 6.3; WOW64)")
+        add("Host", "www.manga-lel.com")
+    }.build()
+
+    private val trustManager = object : X509TrustManager {
```
Comment (inorichi) [inline src/fr/mangalel/src/eu/kanade/tachiyomi/extension/fr/mangalel/Mangalel.kt]: Do you really need this or just copied it from mangahere?

**Comment by inorichi** [comment]
Oh wait, I was reviewing this without reading @null-dev's comment.

The problem with the big CMS package is discoverability. How about we use `Tachiyomi: My Manga Reader CMS (Many sources)` as app name?

**Comment by paronos** [comment]
I think that this PR can be closed.

**Comment by Hugome** [comment]
Oh yes my bad i forget.

**Pull request closed without merging**